drivers/entropy/entropy_gecko_trng.c

```diff
@@ -13,6 +13,7 @@
 
 static void entropy_gecko_trng_read(uint8_t *output, size_t len)
 {
+#ifndef CONFIG_CRYPTO_ACC_GECKO_TRNG		
```

Comment:
I have kept as is because of the following two reasons.
1. CONFIG_CRYPTO_ACC_GECKO_TRNG is defined if SOC_SERIES_EFR32BG22 is defined.
2. If a new board is added which uses TRNG hardware module and not CRYPTO_ACC hardware module, then this has to be changed.

Reply:
In that case the Kconfig setting should be promtless, so it is not visible to the user when executing menuconfig or guiconfig, see https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html?highlight=kconfig%20prompt#id1